<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="listpage">
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="editpage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Birthday</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Subject</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Class</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Sex</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QRadioButton" name="femaleradioButton_2">
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ageEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QDateEdit" name="birthdayEdit"/>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="classGradeComboBox"/>
        </item>
        <item row="5" column="2">
         <widget class="QComboBox" name="classRoomComboBox"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Subject Choice</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Subject Level</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Chinese</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Math</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>English</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="subjectChoice4ComboBox"/>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="subjectChoice5ComboBox"/>
        </item>
        <item row="12" column="1">
         <widget class="QComboBox" name="subjectChoice6ComboBox"/>
        </item>
        <item row="7" column="2">
         <widget class="QComboBox" name="subjectLevel1ComboBox"/>
        </item>
        <item row="8" column="2">
         <widget class="QComboBox" name="subjectLevel2ComboBox"/>
        </item>
        <item row="9" column="2">
         <widget class="QComboBox" name="subjectLevel3ComboBox"/>
        </item>
        <item row="10" column="2">
         <widget class="QComboBox" name="subjectLevel4ComboBox"/>
        </item>
        <item row="11" column="2">
         <widget class="QComboBox" name="subjectLevel5ComboBox"/>
        </item>
        <item row="12" column="2">
         <widget class="QComboBox" name="subjectLevel6ComboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="maleradioButton">
          <property name="text">
           <string>Male</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1" colspan="2">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Discard|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuEntries">
    <property name="title">
     <string>Entries</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
   </widget>
   <addaction name="menuEntries"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
